lib/pages/Contact.dart

```diff
@@ -8,24 +9,280 @@ class Contact extends StatefulWidget {
 class _ContactState extends State<Contact> {
   @override
   Widget build(BuildContext context) {
+    double width = MediaQuery.of(context).size.width;
     return Scaffold(
       appBar: AppBar(
         title: Text('COVID-19 Tracker: Zamboanga City'),
         centerTitle: true,
       ),
-      body: Center(
-        child: Column(
-          mainAxisAlignment: MainAxisAlignment.center,
-          children: <Widget>[
-            Text(
-              'Contact',
-            ),
-            Text(
-              'Coming soon...',
-            ),
-          ],
+      body: Padding(
+        padding: EdgeInsets.fromLTRB(0, 10, 0, 10),
+        child: Center(
+          child: Column(
+            mainAxisAlignment: MainAxisAlignment.start,
+            children: <Widget>[
+              Card(
+                elevation: 5.0,
+                margin: EdgeInsets.fromLTRB(10, 0, 10, 10),
+                child: Padding(
+                  padding: const EdgeInsets.all(15.0),
+                  child: Row(
+                    children: <Widget>[
+                      CircleAvatar(
+                        child: Text( 'ICON',
+                          style: TextStyle(
+                            color: Color(0xff44337a),
+                          ),
+                        ),
+                        radius: 30,
+                        backgroundColor: Theme.of(context).accentColor
+                      ),
+                      Container(
+                        width: width - 120,
+                        child: ListTile(
+                          title: Text( 'HOTLINE NUMBERS',
+                            style: TextStyle(
+                              color: Color(0xffff5a6f),
+                              fontSize: 22
+                            ),
+                          ),
+                          subtitle: Text( 'Contact details from Beng Climaco and DOH RO IX Facebook page.',
```

Comment:
Changed this from `Contact details from Beng Climaco and DOH RO IX Facebook page.` to `Contact details sourced from Beng Climaco, Zamboanga City Medical Center, and DOH RO IX Facebook pages.`